src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj

```diff
@@ -1,16 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<LangVersion>11.0</LangVersion>
+		<LangVersion>12.0</LangVersion>
 		<OutputType>WinExe</OutputType>
-		<TargetFramework>net7.0-windows10.0.22621.0</TargetFramework>
+		<TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
 		<TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
 		<SupportedOSPlatformVersion>10.0.20348.0</SupportedOSPlatformVersion>
 		<RootNamespace>ApplicationTemplate</RootNamespace>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 		<ApplicationManifest>app.manifest</ApplicationManifest>
 		<Platforms>x86;x64;arm64</Platforms>
-		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-		<PublishProfile>win10-$(Platform).pubxml</PublishProfile>
+		<RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+		<SelfContained>true</SelfContained>
```

Comment:
Is this supposed to be here?

Reply:
yes, they made a change to net8, self contained was used by default previously
https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/runtimespecific-app-default

Reply:
Ok, but should we keep SelfContained if it's no longer the default? Is it building faster without it?

Reply:
Actually, let's keep it like this for now to limit the changes. But we should consider staying on the default value.